src/category_theory/cofiltered_system.lean

```diff
@@ -190,6 +264,10 @@ The subfunctor of `F` obtained by restricting to the eventual range at each inde
   map_id' := λ i, by { simp_rw F.map_id, ext, refl },
   map_comp' := λ _ _ _ _ _, by { simp_rw F.map_comp, refl } }
 
+noncomputable instance to_eventual_ranges_fintype [∀ j, fintype (F.obj j)] :
+  ∀ j, fintype (F.to_eventual_ranges.obj j) :=
+λ j, @fintype.of_finite (F.to_eventual_ranges.obj j) subtype.finite
```

Comment:
I don't think we should add noncomputable `fintype` instances in general; if they existed, they should be gradually replaced by `finite`.